modules/oe_translation_corporate_workflow/modules/oe_translation_corporate_workflow_epoetry/oe_translation_corporate_workflow_epoetry.info.yml

```diff
@@ -0,0 +1,8 @@
+name: OpenEuropa Translation Corporate Workflow ePoetry
```

Comment:
I made this submodule because there are certain aspects that we need to override when it comes to ePoetry in the corporate workflow. Most are inherited from the generic remote translator functionality, but some are not: such as access to creating update requests to ongoing translation requests.